.github/workflows/tag.yaml

```diff
@@ -24,18 +24,20 @@ jobs:
     - name: Get Version
       id: get-version
       run: |
-        CURRENT_VERSION=$(awk -F'"' '/ID string =/ {print $2}' ${{ env.VERSION_FILE }})
-        echo "VERSION=$CURRENT_VERSION" >> "$GITHUB_OUTPUT"
+        VERSION=$(awk -F'"' '/ID string =/ {print $2}' ${{ env.VERSION_FILE }})
+        echo "VERSION=$VERSION" >> "$GITHUB_OUTPUT"
     - name: Create Tag
       if: ${{ steps.get-version.outputs.VERSION != '' }}
       run: |
-        if [ $(git tag -l "${{ steps.get-version.outputs.VERSION }}") ]; then
-            echo "Tag already exists for version $CURRENT_VERSION"
+        VERSION=${{ steps.get-version.outputs.VERSION }}
+        if [ $(git tag -l "$VERSION") ]; then
+            echo "Tag already exists for version $VERSION"
             exit 0
         else
+          git config --global tag.gpgsign true
```

Comment:
This is mentioned in the default Gitsign configuration: https://github.com/sigstore/gitsign?tab=readme-ov-file#configuration

However, the Action does not handle this by default:
https://github.com/chainguard-dev/actions/blob/e82b4e5ae10182af72972addcb3fedf7454621c8/setup-gitsign/action.yml#L103-L111

Reply:
Opened a PR to add `git config --global tag.gpgsign true` to the Action: https://github.com/chainguard-dev/actions/pull/442